src/rules/__tests__/no-snapshots.test.js

```diff
@@ -0,0 +1,32 @@
+import { RuleTester } from 'eslint';
+import rule from '../no-snapshots';
+
+const ruleTester = new RuleTester();
+
+const errors = [{ messageId: 'noSnapshots' }];
+
+ruleTester.run('no-snapshots', rule, {
+  valid: [
```

Comment:
Could you chuck in a few more valid tests that might come up during coding?

I'm thinking just things like `expect(a);` and `toMatchSnapshot();`.

I know those are pretty plain and boring tests, but they're super useful in setting the boundaries on what is and isn't expected for a rule, which in turn makes future work easier.